templates/control/edit_searchtagblacklist.html

```diff
@@ -0,0 +1,23 @@
+$def with (tags)
+$:{RENDER("common/stage_title.html", ["Edit Searchtag Blacklist", "Settings"])}
```

Comment:
Here and everywhere else in the templates, just call them “Tags”, please. “Searchtag” is kind of an internal code thing (and it doesn’t make a lot of sense there, either 😜).